AirCastingTests/Logic/BluetoothSession/DefaultStandaloneModeContollerTests.swift

```diff
@@ -20,19 +20,29 @@ final class DefaultStandaloneModeContollerTests: ACTestCase {
         Resolver.test.register { self.activeSessionProvider as ActiveMobileSessionProvidingService }
     }
     
-    func TOBEDONEtest_whenThereIsActiveSession_changesSessionStatusToDisconnected() throws {
+    func test_whenThereIsActiveSession_changesSessionStatusToDisconnected() throws {
```

Comment:
This test fails, should it?
![Zrzut ekranu 2022-12-5 o 12 00 34](https://user-images.githubusercontent.com/44276070/205621304-252d96a1-2a26-4d11-9c66-9298395240d7.png)

Reply:
This PR is still work in progress. Sorry for not adding a label